fdbserver/QuietDatabase.actor.cpp

```diff
@@ -398,9 +398,18 @@ ACTOR Future<TraceEventFields> getStorageMetricsTimeout(UID storage, WorkerInter
 		    wi.eventLogRequest.getReply(EventLogRequest(StringRef(storage.toString() + "/StorageMetrics")));
 		state Future<Void> timeout = delay(30.0);
 		choose {
-			when(TraceEventFields res = wait(result)) {
-				if (version == invalidVersion || getDurableVersion(res) >= static_cast<int64_t>(version)) {
-					return res;
+			when(state TraceEventFields res = wait(result)) {
```

Comment:
I'd recommend using a different, more descriptive variable name for state variables and optionally extracting this to a higher scope

Reply:
done.